src/concuerror_options.erl

```diff
@@ -152,6 +152,13 @@ options() ->
     "                exploration is too slow. Use 'optimal' if a lot of~n"
     "                interleavings are reported as sleep-set blocked.~n"
     "- 'persistent': Using persistent sets. Do not use."}
+  ,{parallel, [experimental], $p, {boolean, false},
+    "Parallel exploration of interleavings",
+    nolong}
+  ,{number_of_processes, [experimental], $n, {integer, 50},
+    "Maximum number of total processes",
+    "The maximum number of different processes that can be spawned from different"
+    "interleavings. Concuerror will stop exploration beyond this limit."}
```

Comment:
Rephrase to user-friendlier language: "The maximum number of processes used by your test."